src/main/java/fr/eilco/mabootique4/panierServlet.java

```diff
@@ -18,34 +24,67 @@
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 
         Panier panier;
+        List<ProduitEntity> panierItemsInfo = new ArrayList<>();
 
         HttpSession session = request.getSession();
         panier = (Panier) session.getAttribute("panier");
+
+
         if(panier == null){
-            session.setAttribute("panier_items", new ArrayList<>());
+            session.setAttribute("panier_items", panierItemsInfo);
         } else {
+
+            try{
+                final Hashtable jndiProperties = new Hashtable();
+                jndiProperties.put(Context.URL_PKG_PREFIXES, "org.jboss.ejb.client.naming");
+                final Context context = new InitialContext(jndiProperties);
+
+                MaBootiqueEJBLocal local = (MaBootiqueEJBLocal)
+                        context.lookup("java:app/MaBootique4-1.0-SNAPSHOT/MaBootiqueJNDI!fr.eilco.mabootique4.ejb.MaBootiqueEJBLocal");
+
+                for(ItemPanier itemPanier : panier.getItems()){
+                    itemPanier.setProduit(local.getProduitById(itemPanier.getProduct_id()));
+                }
+
+            }catch (Exception e) {
+                e.printStackTrace();
+            }
+
+            panier.updateTotalPanier();
+
             session.setAttribute("panier_items", panier.getItems());
+
         }
 
         request.getRequestDispatcher("panier.jsp").forward(request, response);
     }
 
     @Override
-    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-        String id = request.getParameter("product_id");
-        System.out.println("RQST : " + request.getParameter("product_id"));
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+
+        String id_to_add = request.getParameter("product_id_to_add");
+        String id_to_remove = request.getParameter("product_id_to_remove");
```

Comment:
camelCase pour le nom des variables, et snake_case pour les noms des paramètres
```suggestion
        String idToAdd = request.getParameter("product_id_to_add");
        String idToRemove = request.getParameter("product_id_to_remove");
```